assets/app/scripts/services/data.js

```diff
@@ -744,6 +755,10 @@ angular.module('openshiftConsole')
         delete params.hookType;
         template = BUILD_HOOKS_URL;
       }
+      else if (typeof subresource !== 'undefined') {
```

Comment:
style thing, we tend to shortcut variable existence checks as truthiness checks
`else if (subresource)`